test/helpers/policygen/models.go

```diff
@@ -623,8 +624,32 @@ func (t *TestSpec) NetworkPolicyApply() error {
 		helpers.KubectlApply,
 		helpers.HelperTimeout)
 
-	if err != nil {
-		return fmt.Errorf("Network policy cannot be imported prefix=%s: %s", t.Prefix, err)
+	if err == nil {
+		return nil
+	}
+
+	// Some test flakes awaiting for the policy version to being updated
```

Comment:
... test flakes occur while waiting for the policy version to be updated. Sometimes, pods are not scheduled on the node and the policy revision is not updated correctly.

What exactly does "policy is ok" mean? Please operationally define what being "ok" means in reference to a policy.